crates/kitsune_p2p/types/src/config.rs

```diff
@@ -3,8 +3,8 @@
 /// How long kitsune should wait before timing out when joining the network.
 pub const JOIN_NETWORK_TIMEOUT: std::time::Duration = std::time::Duration::from_secs(20);
 
-/// One hour
-pub const RECENT_THRESHOLD_DEFAULT: std::time::Duration = std::time::Duration::from_secs(60 * 60);
+/// Five minutes
```

Comment:
Is this actually long enough for recent gossip to likely reach all online nodes? Maybe 10 or 15 mins is more realistic?

Reply:
@freesig Yeah maybe 15 makes sense? I made this configurable in the tuning params so I can bump it there.